app/handler/commands.js

```diff
@@ -657,6 +658,27 @@ exports.handleMessage = function (steamID, message) {
                 client.chatMessage(steamID, 'Removed "' + entry.name + '".');
             }
         });
+    } else if (isAdmin && command === 'trades') {
+        const dateNow = new Date().getTime(); // Gets date & time in milliseconds
+        const offerData = manager.pollData.offerData
+
+        let tradeCount = 0;
+        let tradeTotal = 0;
+        for (const offerID in offerData) {
+            if (!Object.prototype.hasOwnProperty.call(offerData, offerID)) {
+                continue;
+            }
+
+            if (offerData[offerID].finishTimestamp >= (dateNow - (86400000)) && offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) { 
+                // Sucessful trades accepted in the last 24 hours by bot
+                tradeCount++;
+                tradeTotal++;
+            } else if (offerData[offerID].handledByUs === true && offerData[offerID].isAccepted === true) { 
+                // Successful trades not accepted in the last 24 hours by bot
+                tradeTotal++;
+            }
```

Comment:
Duplicate code. First check if the offer is accepted and handled by us, then increment tradeTotal. If it is within 24 hours then add it to tradeCount